api/app/src/main/kotlin/packit/service/PacketService.kt

```diff
@@ -44,14 +45,21 @@ class BasePacketService(
     private val outpackServerClient: OutpackServer
 ) : PacketService
 {
+    private fun getDisplayNameForPacket(packet: OutpackMetadata): String {
+        val metadata = outpackServerClient.getMetadataById(packet.id)
+        val orderlyMetadata = metadata?.custom?.get("orderly") as? Map<*, *>
+        val description = orderlyMetadata?.get("description") as? Map<*, *>
+        return description?.get("display") as? String ?: packet.name
+    }
+
```

Comment:
I'm surprised you need to do this - is the custom metadata not already in the metadata being returned for all new packets in line 59? If it's not it should be. Maybe it is but those values ujust aren't being passed out by `getMetadata`?  We shouldn't need to do a new call to the client per packet.

Reply:
It does seem inefficient doesn't it. Line 59 corresponds to calling `/packit/metadata` on the outpack server, which returns only id, name, parameters, and time, contrary to the documentation of outpack server: https://github.com/mrc-ide/outpack_server/pull/68

Reply:
So we'll change the outpack server

Reply:
Changed outpack server in https://github.com/mrc-ide/outpack_server/pull/69